neptune_helper.py

```diff
@@ -40,14 +40,18 @@ def iam_connect():
     if neptune_endpoint is None:
         raise EnvironmentVariableNotSetError('NEPTUNE_ENDPOINT')
 
+    neptune_port = getenv('NEPTUNE_PORT', None)
+    if neptune_port is None:
+        neptune_port = 8182
+
     GremlinUtils.init_statics(globals())
     session = Session(aws_access_key_id=access_key,
                       aws_secret_access_key=secret_key,
                       aws_session_token=session_token,
                       region_name=region)
     credentials = session.get_credentials()
     endpoints = Endpoints(neptune_endpoint=neptune_endpoint,
-                          neptune_port=8182,
+                          neptune_port=neptune_port,
```

Comment:
Have you checked what's the type of the value returned by `getenv()`? You might need to do: 

```python
neptune_port=int(neptune_port),
```

Reply:
Your are right!! the getenv() function returns a string, which makes sense, so yes I also have to convert the values to int when passing it to the Endpoints constructor.